src/test/EC3ConfigurationsClient.test.ts

```diff
@@ -0,0 +1,261 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as chaiAsPromised from "chai-as-promised";
+import { expect, use } from "chai";
+import * as sinon from "sinon";
+import { EC3ConfigurationsClient } from "../reporting/clients/EC3ConfigurationsClient";
+import { EC3Configuration, EC3ConfigurationCreate, EC3ConfigurationUpdate } from "../reporting/interfaces/EC3Configurations";
+use(chaiAsPromised);
+
+describe("EC3 Configurations Client", () => {
+  const configurationsClient: EC3ConfigurationsClient = new EC3ConfigurationsClient();
+  const configurationsClientNewBase: EC3ConfigurationsClient = new EC3ConfigurationsClient("BASE");
+  let fetchStub: sinon.SinonStub;
+  let requestStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    fetchStub = sinon.stub(EC3ConfigurationsClient.prototype, "fetchJSON" as any);
```

Comment:
Have you tried stupping the `fetchJSON` method directly on the instance, not on the prototype?

Reply:
stubbed on instance